recipes-extended/xen/files/v4v.patch

```diff
@@ -2651,35 +2638,42 @@ PATCHES
 + */
 --- a/xen/include/public/xen.h
 +++ b/xen/include/public/xen.h
-@@ -118,7 +118,7 @@ DEFINE_XEN_GUEST_HANDLE(xen_ulong_t);
+@@ -118,10 +118,11 @@ DEFINE_XEN_GUEST_HANDLE(xen_ulong_t);
  #define __HYPERVISOR_domctl               36
  #define __HYPERVISOR_kexec_op             37
  #define __HYPERVISOR_tmem_op              38
--#define __HYPERVISOR_xc_reserved_op       39 /* reserved for XenClient */
+-#define __HYPERVISOR_argo_op              39
 +#define __HYPERVISOR_v4v_op               39 /* reserved for XenClient */
  #define __HYPERVISOR_xenpmu_op            40
  #define __HYPERVISOR_dm_op                41
+ #define __HYPERVISOR_txt_op               42
++#define __HYPERVISOR_argo_op              43
```

Comment:
I don't think this change is the correct way to patch in v4v into Xen 4.12: it disables compatibility with Argo, which is the upstream technology. Instead, v4v should be issued the hypercall number 43, leaving Argo in place with 39. The numbering should be updated in the v4v Linux device driver to match the change here.

Reply:
I actually had that originally, but I changed it to what I have now to avoid modifying other repos (e.g. v4v in your example). If that's the desired approach I'll change it back.

Reply:
Yes, the change would be appreciated. Thanks.